src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Export/GroupMemberExtractor.cs

```diff
@@ -106,9 +106,9 @@ private async Task<HashSet<string>> GetGroupPatientIdsHelper(string groupId, Dat
                 if (
                     (member.Inactive == null
                     || member.Inactive == false
-                    || member.Period?.EndElement == null
-                    || member.Period?.EndElement > fhirGroupMembershipTime
                     || includeInactiveMembers)
+                    && (member.Period?.EndElement == null
```

Comment:
What's the expected behavior when includeInactiveMembers = true and the EndElements < fhirGroupMembershipTime? The current condition will exclude that member. But do they count as "inactive" member?

Reply:
I flip-flopped on this. The spec doesn't say one way or the other, so I would say it is up to us. I left it this way to differentiate between the idea of a resource being an inactive member and a resource no longer being in the group.

Reply:
Sounds good to me.